server/src/test/java/org/opensearch/search/approximate/ApproximatePointRangeQueryTests.java

```diff
@@ -256,8 +257,7 @@ public void testApproximateRangeShortCircuitAscSort() throws IOException {
                     for (int v = 0; v < dims; v++) {
                         scratch[v] = i;
                     }
-                    doc.add(new LongPoint("point", scratch));
-                    iw.addDocument(doc);
+                    iw.addDocument(asList(new LongPoint("point", scratch[0]), new NumericDocValuesField("point", scratch[0])));
```

Comment:
Why is this change needed? Is `testApproximateRangeShortCircuitAscSort` also one of the failing test?

Reply:
I ran each test about 1000 times, `testApproximateRangeShortCircuitAscSort` fails about 1% of the time interestingly. I then realized that we had to force the sorting behavior after collecting docIDs since the two different queries are collecting docIds in different order

Reply:
Can we also try running these tests individually until failure and ensure they are not failing before removing awaits annotation?

Reply:
I ran all tests until failure and it kept going until 5000+ iterations, I think I can safely say that there is no flakiness here

Reply:
Thanks @harshavamsi for the due diligence! I was slightly concerned with 1% within 1000 iterations.